src/mixed-line-bar-chart/__integ__/announcements.test.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import createWrapper from '../../../lib/components/test-utils/selectors';
+import { setupTest } from './common';
+
+describe('Popover content is announced as plain text on hover', () => {
+  describe.each(['keys', 'values'])('when using links in %s', useLinks => {
+    const alwaysVisibleTexts = ['Jun 2023,', 'Amazon Simple Storage Service $69.80,', 'Group 1 $83.31,'];
+    const collapsedText = 'AWS Config';
+
+    test(
+      'without expandable sub-items',
+      setupTest(`#/light/mixed-line-bar-chart/drilldown?useLinks=${useLinks}&expandableSubItems=false`, async page => {
+        const wrapper = createWrapper().findMixedLineBarChart();
+        const bar = wrapper.findBarGroups().get(3).toSelector();
+        const getLabel = () => page.getElementAttribute(bar, 'aria-label');
+        await page.hoverElement(bar);
+        await page.waitForAssertion(async () => {
+          const label = await getLabel();
+          for (const text of alwaysVisibleTexts) {
+            expect(label).toContain(text);
+          }
+          // Collapsed sub-items should not be announced
+          expect(label).toContain(collapsedText);
+          expect(label).not.toContain('[object object]');
```

Comment:
Would it be easier and safer to test against the actual value e.g. `expect(label).toBe('....')`

Reply:
Refined the test a bit [here](https://github.com/cloudscape-design/components/pull/1644/commits/7c8c1496906031dec36a1076d338a94be7ba5ff3), added a few more texts and using `toBe` once in line 44. However, the reason to use `toContain` is basically the expand/collapse functionality: when expanding, sub-items are inserted just after their parent and not at the end, so I need to be able compose by picking and choosing elements.